index.js

```diff
@@ -190,9 +199,11 @@ function compression (options) {
 
       // compression stream
       debug('%s compression', method)
-      stream = method === 'gzip'
-        ? zlib.createGzip(opts)
-        : zlib.createDeflate(opts)
+      switch (method) {
+        case 'br': stream = zlib.createBrotliDecompress(opts); break
```

Comment:
I believe this should be `createBrotliCompress`, not `createBrotliDecompress`, right?

Reply:
Thanks 😄